### PR TITLE
pkp/pkp-lib#7105 Job processing via schedule task

### PR DESCRIPTION
**Pull request by touhidurabir**
This PR aims to provide solution for pkp/pkp-lib#7105

### CODE REVIEW

File: registry/scheduledTasks.xml [hunk 1]
```diff
@@ -59,4 +59,8 @@
 		<descr>Process the usage stats logs and load the numbers into the DB tables.</descr>
 		<frequency day="0"/>
 	</task>
+	<task class="lib.pkp.classes.task.ProcessQueueJobs">
+		<descr>Process pending queue jobs in the default queue driver and queue</descr>
+		<frequency minute="0"/>
```
Comment (asmecher) [inline registry/scheduledTasks.xml:64]: We discussed changing this to `second="0"` to get it to run every minute, but at a glance it looks like `lib/pkp/classes/scheduledTask/ScheduledTaskHelper.php` will need to be extended to include that option.

Comment (touhidurabir) [reply registry/scheduledTasks.xml:64]: Done extending .

File: tools/runScheduledTasks.php [hunk 2]
```diff
@@ -19,5 +19,7 @@ class runScheduledTasks extends \PKP\cliTool\ScheduledTaskTool
 {
 }
 
+// app('pkpJobQueue')->runJobsInQueue();
```
Comment (asmecher) [inline tools/runScheduledTasks.php:22]: May as well remove this, since it's handled elsewhere!

File: registry/scheduledTasks.xml [hunk 3]
```diff
@@ -59,4 +59,8 @@
 		<descr>Process the usage stats logs and load the numbers into the DB tables.</descr>
 		<frequency day="0"/>
 	</task>
+	<task class="lib.pkp.classes.task.ProcessQueueJobs">
```
Comment (asmecher) [inline registry/scheduledTasks.xml:62]: Needs changing to: `PKP\task\ProcessQueueJobs`

File: tools/runScheduledTasks.php (hunk 2 above)
Comment (touhidurabir) [reply tools/runScheduledTasks.php]: Done.

File: registry/scheduledTasks.xml (hunk 3 above)
Comment (touhidurabir) [reply registry/scheduledTasks.xml:62]: Done.

**Comment by asmecher** [comment]
Manually merged (submodule conflict)

**Pull request closed without merging**